backend/memory/mailbox.go

```diff
@@ -12,13 +12,23 @@ import (
 var Delimiter = "/"
 
 type Mailbox struct {
-	Subscribed bool
-	Messages   []*Message
+	Subscribed  bool
+	Messages    []*Message
+	UidValidity uint32
 
 	name string
 	user *User
 }
 
+func NewMailbox(user *User, name string) *Mailbox {
+	mbox := &Mailbox{
+		name: name, user: user,
+		UidValidity: uint32(time.Now().Nanosecond()),
```

Comment:
This doesn't work, 2^32 nanoseconds is [just a few seconds](https://www.wolframalpha.com/input/?i=2%5E32+nanoseconds).

Reply:
Already changed back to 1 because of the tests.